docs/listings/error_handling/Scarb.toml

```diff
@@ -6,10 +6,10 @@ edition = "2024_07"
 # See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
 
 [dependencies]
-snforge_std.workspace = true
-sncast_std.workspace = true
-starknet.workspace = true
-assert_macros.workspace = true
```

Comment:
What's the reason for this change? Applies also to other similar changes.

Reply:
We don't have workspaces in listings now (apart from `hello_workspaces` package which is workspace for docs purposes).